semcore/product-head/src/style/title.shadow.css

```diff
@@ -9,6 +9,7 @@ STitle {
   text-overflow: ellipsis;
   white-space: nowrap;
   color: var(--intergalactic-text-primary, #191b23);
+  margin: 0;
```

Comment:
Probably it was noticed before because many services use css reset on their pages while our website doesn't.